src/main/scala/nl.knaw.dans.easy.deposit/DataFiles.scala

```diff
@@ -34,6 +35,28 @@ import scala.util.{ Failure, Success, Try }
  */
 case class DataFiles(bag: DansBag) extends DebugEnhancedLogging {
 
+  private val dataDir = bag.baseDir / "data"
```

Comment:
This is the same as `bag.data`. I prefer the latter. so drop this one.

Reply:
@vesaakerman @jo-pol It is not a big deal to me, but there seems little value in this field. Any arguments in favor or contra (apart from preferences)?

Reply:
The field used to be valuable before the new bag-lib was used. Now that we also have `bag.data` it is confusing to use both. Only by this review I saw I should have cleaned it up when introducing the bag-lib.